DNN Platform/DotNetNuke.Abstractions/Portals/IPortalInfo.cs

```diff
@@ -0,0 +1,310 @@
+        /// <value>TabdId of the Search Results page.</value>
+        int SearchTabId { get; set; }
+
+        /// <summary>
+        /// Gets or sets tabid of the Custom 404 page.
+        /// </summary>
+        /// <value>Tabid of the Custom 404 page.</value>
+        int Custom404TabId { get; set; }
+
+        /// <summary>
+        /// Gets or sets tabid of the Custom 500 error page.
+        /// </summary>
+        /// <value>Tabid of the Custom 500 error page.</value>
+        int Custom500TabId { get; set; }
+
+        /// <summary>
+        /// Gets or sets tabid of the Terms of Use page.
+        /// </summary>
+        /// <value>Tabid of the Terms of Use page.</value>
+        int TermsTabId { get; set; }
+
+        /// <summary>
+        /// Gets or sets tabid of the Privacy Statement page.
+        /// </summary>
+        /// <value>Tabid of the Privacy Statement page.</value>
+        int PrivacyTabId { get; set; }
+
+        /// <summary>
+        /// Gets or sets tabdId of the splash page. If 0, there is no splash page.
+        /// </summary>
+        /// <value>TabdId of the Splash page.</value>
+        int SplashTabId { get; set; }
+
+        /// <summary>
+        /// Gets or sets tabId at which Host tasks start.
+        /// </summary>
+        /// <value>TabId of Host tasks.</value>
+        int SuperTabId { get; set; }
+
+        /// <summary>
+        /// Gets or sets number of registered users allowed in the portal. 0 means not limited.
+        /// </summary>
+        /// <value>Number of registered users allowed. </value>
+        int UserQuota { get; set; }
+
+        /// <summary>
+        /// Gets or sets type of registration that the portal supports.
+        /// </summary>
+        /// <value>Type of registration.</value>
+        int UserRegistration { get; set; }
+
+        /// <summary>
+        /// Gets or sets tabid of the User profile page.
+        /// </summary>
+        /// <value>TabdId of the User profile page.</value>
+        int UserTabId { get; set; }
+
+        /// <summary>
+        /// Gets or sets actual number of actual users for this portal.
+        /// </summary>
+        /// <value>Number of users for the portal.</value>
+        int Users { get; set; }
+
+        /// <summary>
+        /// Gets or sets dNN Version # of the portal installation.
+        /// </summary>
+        /// <value>Version # of the portal installation.</value>
+        string Version { get; set; }
+
+        /// <summary>
+        /// Gets or sets the actual name of the Administrators group of the portal.
+        /// This name is retrieved from the RoleController object.
+        /// </summary>
+        /// <value>The name of the Administrators group.</value>
+        string AdministratorRoleName { get; set; }
+
+        /// <summary>
+        /// Gets or sets actual number of pages of the portal.
+        /// </summary>
+        /// <value>Number of pages of the portal.</value>
+        int Pages { get; set; }
```

Comment:
@mitchelsellers as above, this seems like an opportunity to not include this property on the interface, in an effort to avoid requiring an `IPortalInfo` implementation to take a dependency on `ITabController` (or `ITabService` or whatever)

Reply:
Agreed